.github/workflows/storybook.yml

```diff
@@ -147,3 +147,33 @@ jobs:
           git pull --no-edit --quiet
           git commit -m "Remove storybook for pull/${PULL_REQUEST_NUMBER}."
           git push origin gh-pages
+
+  test-storybook:
+    name: Test Storybook
+    runs-on: ubuntu-latest
+    needs: build-storybook
```

Comment:
Previously the new storybook test passed using this. Now it's failing because it doesn't have the built output :thinking: